front/src/applications/stdcm/components/StdcmResults/SendToRailwayManagerModal.tsx

```diff
@@ -0,0 +1,430 @@
+import { useMemo, useState, useRef, useCallback } from 'react';
+
+import { Button, Input, Select, TextArea } from '@osrd-project/ui-core';
+import { Download, CheckCircle } from '@osrd-project/ui-icons';
+import { PDFDownloadLink, pdf } from '@react-pdf/renderer';
+import { useTranslation } from 'react-i18next';
+
+import {
+  type StdcmSimulationInputs,
+  type StdcmSuccessResponse,
+  type SimilarTrainWithSecondaryCode,
+  type StdcmResultsOperationalPoint,
+} from 'applications/stdcm/types';
+import {
+  osrdRailwayManagerApi,
+  type BodyLastMinuteRequest,
+} from 'common/api/osrdRailwayManagerApi';
+import MAIN_API from 'config/config';
+import {
+  createLinkedTrainPayload,
+  createSimilarTrainPayload,
+  getStopDurationTime,
+  transformStepsToApiFormat,
+  validateTolerance,
+} from 'modules/SimulationReportSheet/utils/formatSimulationReportSheet';
+import type { StdcmPathStep } from 'reducers/osrdconf/types';
+import { useDateTimeLocale } from 'utils/date';
+import type { DeploymentSettings } from 'utils/hooks/useDeploymentSettings';
+import { kmhToMs, minToMs, msToMin, tToKg } from 'utils/physics';
+
+import StdcmSimulationReportSheet from './StdcmSimulationReportSheet';
+
+type SendToRailwayManagerModalProps = {
+  onClose: () => void;
+  consist: StdcmSimulationInputs['consist'];
+  stdcmData: StdcmSuccessResponse;
+  linkedTrains: StdcmSimulationInputs['linkedTrains'];
+  simulationReportSheetNumber: string;
+  operationalPointsList: StdcmResultsOperationalPoint[];
+  simulationSheetLogo?: string;
+  similarTrains: SimilarTrainWithSecondaryCode[];
+  deploymentSettings?: DeploymentSettings;
+};
+
+type Option = { value: string; label: string };
+
+const SendToRailwayManagerModal = ({
+  consist,
+  onClose,
+  stdcmData,
+  linkedTrains,
+  simulationReportSheetNumber,
+  operationalPointsList,
+  simulationSheetLogo,
+  similarTrains,
+  deploymentSettings,
+}: SendToRailwayManagerModalProps) => {
+  const { t } = useTranslation('stdcm', { keyPrefix: 'simulation.results' });
+  const dateTimeLocale = useDateTimeLocale();
+
+  const steps = stdcmData?.simulationPathSteps ?? [];
```

Comment:
`stdcmData` is always defined.
```suggestion
  const { simulationPathSteps: steps } = stdcmData;
```

Reply:
Done